cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -1553,6 +1564,14 @@ instance ToJSON PeerSelectionTargets where
                  , "targetActiveBigLedgerPeers" .= nActiveBigLedgerPeers
                  ]
 
+instance ToJSON peerAddr => ToJSON (PublicRootPeers peerAddr) where
+  toJSON prp =
+    Aeson.object [ "kind" .= String "PublicRootPeers"
```

Comment:
We often do the `kind` thing, but it would be more in the spirit of `json` to communicate this through the field name that includes the record.  But that would be a large change and I am not sure if it's worth the trouble.